lib/oban/telemetry.ex

```diff
@@ -205,6 +205,26 @@ defmodule Oban.Telemetry do
 
   defp duration(start_mono), do: System.monotonic_time() - start_mono
 
+  @doc false
+  def execute(event_name, measurements, meta) do
+    :telemetry.execute(event_name, measurements, normalize_meta(meta))
+  end
+
+  defp normalize_meta(%{name: {:via, Registry, {Oban.Registry, {_pid, name}}}} = meta) do
```

Comment:
It took me a little reading through the telemetry docs to realize where the `name` is even used. Nice catch 👍

Reply:
It was caught by the tests :-) Good job on making them exhaustive :+1: